kroxylicious-filter-test-support/src/main/java/io/kroxylicious/test/assertj/HeaderAssert.java

```diff
@@ -28,13 +34,13 @@ public static HeaderAssert assertThat(Header actual) {
     private AbstractStringAssert<?> key() {
         var existingDescription = descriptionText();
         return Assertions.assertThat(actual.key())
-                .describedAs(existingDescription + " key");
+                .describedAs(existingDescription + KEY_SUFFIX);
     }
 
     public AbstractByteArrayAssert<?> value() {
```

Comment:
Sonar is whinging about this. I think we should suppress that warning, because really we're just following the pattern that assertj has made use of, so it's not really our problem.